server/server.js

```diff
@@ -122,6 +130,23 @@ var tfamatFile = {
 app.post('/genotet', function(req, res) {
   var type = req.body.type;
   console.log('POST', type);
+
+  switch(type) {
+    // upload
+    case 'upload':
+      var fileType = req.query.type;
```

Comment:
If I recall correctly, post seems to only work with req.body and does not have a query attribute set. Have you tested this to see if it works?